lib/configs/createStyleguideConfig.js

```diff
@@ -189,42 +189,6 @@ module.exports = (config = {}, options = {}) => {
 
     // only the root should alias singletons or check circularity
     if (isRootConfig()) {
-        // IE 11 support for styleguidist-generated artifacts
-        // https://github.com/styleguidist/react-styleguidist/pull/1327#issuecomment-483928457
-        const ie11ModuleTransforms = [
-            'acorn-jsx',
-            'estree-walker',
-            'regexpu-core',
-            'unicode-match-property-ecmascript',
-            'unicode-match-property-value-ecmascript',
-            'react-dev-utils',
-            'ansi-styles',
-            'ansi-regex',
-            'chalk',
-            'strip-ansi',
-            ...(options.ie11ModuleTransforms || []),
```

Comment:
Do you want to delete the documentation about this `ie11ModuleTransforms` option?

Reply:
https://github.com/zillow/create-react-styleguide#createstyleguideconfigconfig-options

Reply:
Great catch! Removed.

Reply:
You might also want to update your commit with notes about dropping that option for the CHANGELOG.